src/test/java/core/basesyntax/service/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+
+    @Test
+    public void readData_emptyPath_notOk() {
+        try {
+            new CsvFileReaderImpl().readData("");
```

Comment:
you should create a global variable for this object and call it everywhere where you use it